packages/cli/lib/lib/webpack/prerender.js

```diff
@@ -30,7 +30,6 @@ module.exports = function(env, params) {
 		const renderToString = require(require.resolve(
 			`${cwd}/node_modules/preact-render-to-string`
 		));
-
 		return renderToString(preact.h(app, { ...params, url }));
```

Comment:
I think the behaviour differs between client side in the entry above and prerender here

Reply:
params has everything coming from https://github.com/preactjs/preact-cli/pull/839/files#diff-b26e9c1b1185752601692f2207489852R17

Reply:
I wish this was typescript 🙃.

Reply:
oh, ok